### DISCUSSION TITLE
Wonderful User Experience

### DISCUSSION THREAD

**Discussion by deanspost** [koreader/koreader#10761] [General]
I cannot express how satisfying the user experience is in KOReader. Every reading improvement I want has already been thought of and I just need to find it in the menu. Absolutely wonderful!!
Btw, I use a Kobo Elipse for reading text books and science papers. KOReader transforms the Elipse from being just ok, to perfect for this purpose.
Well done :)

Side note, I don't see anywhere to express my satisfaction (like an upvote button) or see other peoples ratings. 
Am I missing something?

**Comment by Frenzie** [comment]
There are stars, but other than that not really.

**Comment by HammyHavoc** [comment]
Just chiming in to say KOReader is excellent and I appreciate the ongoing work, care and passion that goes into it. Wishing you and yours excellent health and happiness.
